brontide/fuzz_test.go

```diff
@@ -59,43 +59,35 @@ var (
 // and completes the brontide handshake between them. If any part of the
 // handshake fails, this function will panic.
 func completeHandshake(t *testing.T, initiator, responder *Machine) {
-	t.Helper()
-
-	if err := handshake(initiator, responder); err != nil {
-		dumpAndFail(t, initiator, responder, err)
-	}
-}
-
-// handshake actually completes the brontide handshake and bubbles up
-// an error to the calling function.
-func handshake(initiator, responder *Machine) error {
```

Comment:
I think the `t.Helper()` actually needed to be called in this scope, but fine to just in line it all.